docs/community_snippets.md

```diff
@@ -32,6 +32,7 @@ _To contribute, check out our [guide here](#contributing)._
 | [PSCustomObject](#pscustomobject) | _A simple PSCustomObject by @brettmillerb_ |
 | [Region Block](#region-block) | _Region Block for organizing and folding of your code_ |
 | [Send-MailMessage](#send-mailmessage) | _Send an mail message with the most common parameters by @fullenw1_ |
+| [New Azure Resource Group](#new-azure-resource-group) | _Create an Azure Resource group by @vmsilvamolina_ |
```

Comment:
This is an alpha sorted list. Can you move it, and the snippet, up to align with that?

Reply:
Fixed it!